sonar-rust-plugin/src/main/java/org/sonarsource/rust/clippy/ClippySensor.java

```diff
@@ -68,6 +69,11 @@ public void execute(SensorContext context) {
       return;
     }
 
+    boolean offlineMode = context.config().getBoolean(CLIPPY_OFFLINE).orElse(false);
```

Comment:
```suggestion
    var offlineMode = context.config().getBoolean(CLIPPY_OFFLINE).orElse(false);
```

Reply:
it will show an SQ issue, because `var` will be inferred to be wrapper type `Boolean` and that should not be used